mindmeld/app_manager.py

```diff
@@ -197,7 +199,8 @@ def parse(
 
         allowed_intents, nlp_params, dm_params = self._pre_nlp(params, verbose)
         processed_query = self.nlp.process(
```

Comment:
```
language = nlp.params.pop('language', self.language)
```

Reply:
As we discussed, we should use the overwriting method stated in my comment